### PR TITLE
Feat/collabsection

### PR DESCRIPTION
**Pull request by ayushhunt**
## Release Notes

- **New Features**
	- Introduced a new `ContributorsGrid` component to the About page, displaying contributor information dynamically.
	- Added an API endpoint for fetching contributor data.
	- Implemented a scheduled task to publish contributor data from a GitHub repository.

- **Improvements**
	- Enhanced the About page layout with contributor details.
	- Updated configuration to allow images from GitHub avatars.

- **Bug Fixes**
	- Removed unnecessary ESLint and TypeScript settings from configuration files.

- **Chores**
	- Updated dependency versions for improved performance and stability.

### CODE REVIEW

File: packages/crypt/tsconfig.tsbuildinfo
Comment (SkidGod4444) [inline packages/crypt/tsconfig.tsbuildinfo:1]: add this file to git ignore!

**Review by SkidGod4444 — CHANGES_REQUESTED** [review]
@ayushhunt Make the changes as requested along with code rabbit bot!

File: apps/www/app/api/getDummyData.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { cache } from '@repo/cache';
+
+type DummyData = {
+  timestamp: string;
+  message: string;
+};
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
+  try {
+    const rawData = await cache.lrange("dummy-data:history", 0, -1);
```
[the remark this answers is not in the document]
Comment (ayushhunt) [reply apps/www/app/api/getDummyData.ts:11]: yes

[the remark this answers is not in the document]
Comment (SkidGod4444) [reply apps/www/app/api/getDummyData.ts:11]: > yes

what you mean by dummy data? 
why it is a history ?

[the remark this answers is not in the document]
Comment (ayushhunt) [reply apps/www/app/api/getDummyData.ts]: This was some test code i will remove them in next commit

File: apps/triggers/src/collaborator.status.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      // Prepare data: list of { login, id, avatar_url, html_url }
+      const contributorData = filteredContributors.map((contributor) => ({
+        login: contributor.login,
+        id: contributor.id,
+        avatar_url: contributor.avatar_url,
+        html_url: contributor.html_url,
+      }));
+
+      // Store data in Redis under a fixed key
+      const redisKey = 'contributors';
```
Comment (SkidGod4444) [inline apps/triggers/src/collaborator.status.ts:67]: What is redis key?

File: packages/crypt/tsconfig.tsbuildinfo
Comment (SkidGod4444) [inline packages/crypt/tsconfig.tsbuildinfo:1]: Why are we still having this file on your pr?

**Review by SkidGod4444 — CHANGES_REQUESTED** [review]
@ayushhunt 2 more changes along with code rabbit warnings!

File: packages/crypt/tsconfig.tsbuildinfo
Comment (ayushhunt) [reply packages/crypt/tsconfig.tsbuildinfo:1]: i have added it to gitignore

**Pull request merged**